### PR TITLE
Register amila.is-a.dev

### PR DESCRIPTION
**Pull request by oxamila**
Added `amila.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

File: domains/amila.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Y-AmilaNirmal",
+        "email": "yamilanirmal@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "https://y-amilanirmal.github.io/y-amilanirmal"
```
Comment (DEV-DIBSTER) [inline domains/amila.json:8]: Invalid CNAME. CNAME should not contain slashes, protocols, or ://.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Review by wdhdev — APPROVED** [review]

File: domains/amila.json (hunk 1 above)
Comment (y-amila-nirmal) [reply domains/amila.json]: yesterday  I  did change my user name Y-AmilaNirmal to y-amila-nirmal so is if you can changed my json details to that

{
    "owner": {
        "username": "y-amila-nirmal",
        "email": "yamilanirmal@gmail.com"
    },

    "record": {
        "CNAME": "https://y-amila-nirmal.github.io/y-amila-nirmal"
    }
}

I am grateful to you. Consider this a little. 🙂

Comment (WilliamDavidHarrison) [reply domains/amila.json]: @y-amila-nirmal You will need to open a new PR updating your file.

Comment (y-amila-nirmal) [reply domains/amila.json]: "You will need to open a new PR updating your file." <- I didn't that, but its working now thank you for help

**Pull request merged**